grammers/TEMParser.g4

```diff
@@ -8,7 +8,11 @@ options {
     tokenVocab=TEMLexer;
 }
 
-program: ( page | controller)*;
+program: big_component*;
+
+
+big_component : page controller ;
```

Comment:
Naming in parser should be camel-cased "bigComponent"
and I would rather call it "entity" instead of "big component"

Reply:
that "s good
there is no problem a bout the name
the important thing is the idea of this rule